README.md

```diff
@@ -548,6 +548,16 @@ $ bin/rails generate maintenance_tasks:install
 
 This ensures that new migrations are installed and run as well.
 
+**What if I've rolled up previous Maintenance Task migrations?**
```

Comment:
I'm not sure about the wording. At Shopify we don't actually roll them up so much as delete them, and rely on the schema. Maybe just:
```suggestion
**What if I've deleted my previous Maintenance Task migrations?**
```